classes/shop/ShopUrl.php

```diff
@@ -189,6 +189,34 @@ public static function getShopUrls($idShop = false)
     }
 
     /**
+     * Get the default shop URL, which is the first defined main URL.
+     *
+     * @return bool|ShopUrl ShopUrl object or false on failure to find one.
+     *
+     * @since   1.1.0
+     * @version 1.1.0 Initial version
+     */
+    public static function getDefaultUrl()
+    {
+        global $shopUrlConfig;
```

Comment:
Well, this global variable was introduced with the previous pull request already. One can change this, of course, but if so, not only in this place. Reasons why I choosed to use a global include:

- I was keen on good caching performance. Content of this global is set by an include_once(), so it's well cacheable code, maximizing performance.
- When starting with class ObjectFileModel I was pretty unsure how well I could encapsulate that stuff, so I started with that global. And left it this way, because it works well enough (for now).
- I'm not aware of a better solution. Which doesn't mean much, because I'm not exactly experienced with PHP and coding for good PHP performance. My most used language is plain C for embedded devices, where we have a compiler.

If somebody can show me on how to to place this variable better, I'm all ears. Code is in ShopUrl.php and ObjectFileModel.php. Showing, like showing actual code, please :-)